frontend/apps/reader/modules/readersearch.lua

```diff
@@ -319,7 +319,6 @@ function ReaderSearch:onShowFulltextSearchInput(search_string)
 
     UIManager:show(self.input_dialog)
     self.input_dialog:onShowKeyboard()
-    return true
```

Comment:
Actually I somewhat overlooked this, but how does it relate to the PR precisely?

Reply:
Right, ever mind.

> Reverts a previous addition of a now unnecessary return statement from the ReaderSearch:onShowFulltextSearchInput(search_string) function in frontend/apps/reader/modules/readersearch.lua.

Reply:
For more context, that was the return statement that was added to stop the bleeding/propagation of a key press to the search box, you might remember that. Since we need that propagation now, it's unnecessary...